docs/troubleshooting/rancher_prerequisite.md

```diff
@@ -0,0 +1,50 @@
+# Prerequisites to run CStor-CSI in rancher based clusters
+
+## Intro
+CStor-CSI provides block volume support through the iSCSI protocol. Therefore, the iSCSI client(initiator) presence on all Kubernetes nodes is required.
+
+## Step to be performed on Rancher based K8s cluster
+
+- If you are using RancherOS as the operating system, you need to enable the iSCSI service and start iSCSI service on all the worker nodes.
+- If you are using Ubuntu or RHEL as the operating system, you need to
+    - Verify if iSCSI initiators are installed on all nodes.
+    - Add the extra_binds under Kubelet service in cluster YAML file to mount the iSCSI binary and configuration inside the kubelet.
+
+### iSCSI services On RancherOS
+To run iSCSI services, execute the following commands on each of the cluster hosts or nodes.
+```sh
+sudo ros s enable open-iscsi
+sudo ros s up open-iscsi
+```
+Run the below commands on all the nodes to make sure the below directories are persistent, by default these directories are ephemeral.
+```sh
+ros config set rancher.services.user-volumes.volumes  [/home:/home,/opt:/opt,/var/lib/kubelet:/var/lib/kubelet,/etc/kubernetes:/etc/kubernetes,/var/openebs]
+system-docker rm all-volumes
+reboot
+```
+
+### iSCSI services on RHEL or Ubuntu
+
+#### Step1:  Verify iSCSI initiator is installed and services are running
+
+Below commands are required to verify and install iscsi services on nodes
```

Comment:
As these iscsi installation and verification steps are common for all of them can we add one generic README with these steps as well .

Reply:
Doing it in a separate PR will be good.

Reply:
(Or) I will copy that tabular column after prerequisites.

Reply:
updated